Core/NoOpRunner.Core.Tests/FactoryProducerTests.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Xunit;
+using Shouldly;
+using NoOpRunner.Core.Shapes.ShapeFactories;
+
+namespace NoOpRunner.Core.Tests
+{
+    public class FactoryProducerTests
+    {
+        [Fact]
+        public void GetFactory_WhenValidBoolGiven_ShouldReturnAppropriateFactory()
+        {
+            // Arrange
+            var expected = new ImpassableShapeFactory();
+            var actual = FactoryProducer.GetFactory(passable: false);
+
+            // Act
```

Comment:
Amm, the "act" steps starts when you call the method that is being tested. So `var actual = FactoryProducer.GetFactory(passable: false);` should be under act, not arrange. The Act step will never be empty, that would imply that your test does nothing. Same in a few other places.

Reply:
Fixed